docs/conceptual/More-about-how-ROCm-uses-PCIe-Atomics.rst

```diff
@@ -1,156 +1,80 @@
 .. meta::
    :description: How ROCm uses PCIe atomics
-   :keywords: PCIe, PCIe atomics, atomics, BAR memory, AMD, ROCm
+   :keywords: PCIe, PCIe atomics, atomics, Atomic operations, AMD, ROCm
 
 *****************************************************************************
 How ROCm uses PCIe atomics
 *****************************************************************************
+AMD ROCm is an extension of the Heterogeneous System Architecture (HSA). To meet the requirements of an HSA-compliant system, ROCm supports queuing models, memory models, and signaling and synchronization protocols. ROCm can perform atomic Read-Modify-Write (RMW) transactions that extend inter-processor synchronization mechanisms to I/O devices starting from Peripheral Component Interconnect Express 3.0 (PCIe™ 3.0). It supports the defined HSA capabilities for queuing and signaling memory operations. To learn more about the requirements of an HSA-compliant system, see the 
+`HSA Platform System Architecture Specification <http://hsafoundation.com/wp-content/uploads/2021/02/HSA-SysArch-1.2.pdf>`_.
 
-ROCm PCIe feature and overview of BAR memory
-================================================================
+ROCm uses platform atomics to perform memory operations like queuing, signaling, and synchronization across multiple CPU, GPU agents, and I/O devices. Platform atomics ensures that atomic operations run synchronously without interruptions or conflicts across multiple shared resources. Similarly, it enables memory alignment and synchronization of processing data within the shared memory locations and Input/Output (I/O) devices.  This sections details how ROCm uses platform atomics and features of PCIe 3.0.
 
-ROCm is an extension of HSA platform architecture, so it shares the queuing model, memory model,
-signaling and synchronization protocols. Platform atomics are integral to perform queuing and
-signaling memory operations where there may be multiple-writers across CPU and GPU agents.
+Platform atomics in ROCm
+==============================
+Platform atomics enables the set of atomic operations that perform RMW actions across multiple processors, devices, and memory locations to execute synchronously without interruption. An atomic operation is a sequence of computing instructions executed as a single, indivisible unit. These instructions are completed in their entirety without any interruptions or are not executed. These operations support 32-bit and 64-bit address formats.
```

Comment:
```suggestion
Platform atomics enable the set of atomic operations that perform RMW actions across multiple processors, devices, and memory locations so that they execute synchronously without interruption. An atomic operation is a sequence of computing instructions run as a single, indivisible unit. These instructions are completed in their entirety without any interruptions. If the instructions can't be completed as a unit without interruption, none of the instructions are run. These operations support 32-bit and 64-bit address formats.
```

I forgot that we should use "run" instead of "executed". Also I added a clarifying statement. It's not perfect, but I felt it needed to be there. You can choose to leave it or fix it or remove it.